ckan/freeze-requirements.sh

```diff
@@ -3,17 +3,10 @@
 set -o errexit
 set -o pipefail
 
-venv=$(mktemp -d)
+pip3 install pipenv
```

Comment:
I don't think we want this file anymore, but we should use the standard pipenv pinning and upgrading process